internal/runners/artifacts/artifacts.go

```diff
@@ -76,6 +78,8 @@ type structuredArtifact struct {
 	URL  string `json:"url"`
 }
 
+const DefaultTarget = "__all__"
```

Comment:
nit: It might be worth commenting where this value comes from, as we're in a runner and not in the buildplanner package or whatever else may help shed light on this value.

Actually, now that I think about it, maybe this constant should live somewhere in the runtime package and we refer to it in this file?